behave/features/specialist_provider.feature

```diff
@@ -34,4 +34,66 @@ Scenario: Specialist Provider Accepts a case
   And I can see a 'Case accepted successfully' message
   When I return to the specialist provider cases dashboard page
   And I select the Accepted tab
-  Then I can see my accepted case reference number
\ No newline at end of file
+  Then I can see my accepted case reference number
+
+@specialist-provider-accept-case-legal-help-form
+Scenario: Specialist Provider Accepts a case
+  Given that I am viewing a case that I have accepted as a specialist provider
+  And I select the Legal help form
+  And The legal help form Your details section has the values
```

Comment:
This and the following step are very similar and you could reuse code by passing in the name of the section heading as a variable.

Reply:
I have updated the feature file to pass the section name as a parameter.